src/models/pieChart.js

```diff
@@ -185,21 +178,20 @@ nv.models.pieChart = function() {
     //------------------------------------------------------------
 
     pie.dispatch.on('elementMouseover.tooltip', function(evt) {
-        tooltip.html(tooltipContent(evt))
-            .style('display', 'inline-block');
+        evt['series'] = {
+            key: evt.data.key,
+            value: evt.data.y,
+            color: evt.color
+        };
+        tooltip.data(evt).hidden(false);
     });
 
     pie.dispatch.on('elementMouseout.tooltip', function(evt) {
-        tooltip.style('display', 'none');
+        tooltip.hidden(true);
     });
 
     pie.dispatch.on('elementMousemove.tooltip', function(evt) {
-        tooltip.style('top', (event.pageY-15)+"px").style("left",(event.pageX+10)+"px");
-    });
-
-    pie.dispatch.on('elementClick.tooltip', function(evt) {
-        tooltip.html(tooltipContent(evt))
-            .style('display', 'inline-block');
+        tooltip.position({top: d3.event.pageY, left: d3.event.pageX})();
```

Comment:
I would definitely check this in other browsers, especially IE.  event.pageY seems to have different standards across browsers.